dftimewolf/lib/processors/turbinia_artifact.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Processes a directory of artifacts with Turbinia."""
+
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+
+from turbinia import evidence
+from turbinia import TurbiniaException
+
+from dftimewolf.lib.containers import containers
+from dftimewolf.lib.modules import manager as modules_manager
+from dftimewolf.lib.processors.turbinia_gcp import TurbiniaProcessorBase
+
+# pylint: disable=no-member
+
+class TurbiniaArtifactProcessor(TurbiniaProcessorBase):
+  """Processes Google Cloud (GCP) disks with Turbinia.
+
+  Attributes:
+    directory_path (str): Name of the directory to process.
+  """
+
+  def __init__(self, state, critical=False):
+    """Initializes a Turbinia Google Cloud (GCP) disks processor.
+
+    Args:
+      state (DFTimewolfState): recipe state.
+      critical (Optional[bool]): True if the module is critical, which causes
+          the entire recipe to fail if the module encounters an error.
+    """
+    super(TurbiniaArtifactProcessor, self).__init__(state, critical=critical)
+    self.directory_path = None
+
+  # pylint: disable=arguments-differ
+  def SetUp(
+      self, directory_path, project, turbinia_zone, sketch_id, run_all_jobs):
+    """Sets up the object attributes.
+
+    Args:
+      directory_path (str): Name of the directory to process.
+      project (str): Name of the GCP project containing the disk to process.
```

Comment:
Isn't this the project where Turbinia lives?

Reply:
Yeah.  This can actually be refactored out for this module.  I removed the attributes for now, but I'll take a second pass to refactor this a bit in this PR before submitting it.